packages/xo-server-test/sample.config.toml

```diff
@@ -11,6 +11,7 @@
 
 [vms]
   default = ''
+  vmToBackup = ''
```

Comment:
We can let it optional. What do you think ?

Reply:
With your previous solution, I think it's a good solution :+1: